### ISSUE TITLE
New tabs set to open "after active tab" are opened in reverse order

### ISSUE THREAD

**Issue by rieje** [mbnuqw/sidebery#1577]
### Steps to reproduce

- Open Firefox in a new profile
- Install Sidebery
- Change the setting "Place new tab (general rule)" from the default "panel end" to "after active tab"

### Actual behavior

Given list of urls to open as tabs via [URLs List](https://addons.mozilla.org/en-US/firefox/addon/urls-list/), they are opened in reverse order when new tabs are set to open "after active tab". With the default setting of "panel end", they are opened in order as expected. I configured Tree Style Tabs addon to have the same behavior (of positioning new tabs to open after active tab) but it doesn't produce this unintended behavior.

### Expected behavior

It should open the tabs in order after the active tab, just like it does when new tabs are configured too open at "panel end" (the default).

P.S. Is it possible to open a list of tabs in its own tree? TST defaults to prompting user whether it should be opened in a tree or as individual tabs. I almost always prefer the former and it's extremely convenient.

### System

Arch Linux (Wayland)

### Firefox version

124.0.2

### Sidebery version

5.2.0

**Comment by mbnuqw** [comment]
Not a bug: [URLs List](https://addons.mozilla.org/en-US/firefox/addon/urls-list/) opens 'First tab' > Sidebery moves it after active tab, URLs List opens 'Second tab' > Sidebery moves it after active tab (so the 'Second tab' appears before the 'First tab') and so on...

> Is it possible to open a list of tabs in its own tree?

If these tabs were open not by Sidebery, no. To get it working, Sidebery needs to postpone new tabs position handling - wait for multiple tabs to get opened and only after some timeout move these tabs to configured position. Technically possible, but it's the whole new way of handling the new tabs and I personally don't need it, so most likely not soon.

**Comment by rieje** [comment]
Since it's not a bug, do you mind if I open feature request to gauge interest? 

* Feature request: smarter handling of multiple tabs when "position new tab" rule is "panel start", "after active tab", and "first child of active tab". When given a list of URLs to open, it is desirable to open them in order from first to last, i.e. top to bottom for vertical tabs. As a result of the these rules for opening tabs, list of tabs are currently opened in reversed order since each tab is opened one-by-one following the new tab rule. [Tree Style Tabs](https://addons.mozilla.org/en-US/firefox/addon/tree-style-tab/) supports this feature so I'm thinking it is useful and/or can be used as reference.

**Comment by mbnuqw** [comment]
> Since it's not a bug, do you mind if I open feature request to gauge interest?

Sure
